soql-analyzer/src/main/scala/com/socrata/soql/analyzer2/FoundTables.scala

```diff
@@ -108,9 +108,9 @@ final case class FoundTables[MT <: MetaTypes] private[analyzer2] (
   private def parentedGraph(parent: ResourceName, soql: BinaryTree[ast.Select]): QueryGraph[MT] = {
     val scopedParent = ScopedResourceName(initialScope, parent)
     val seed = Map(
-      Option.empty[types.ScopedResourceName[MT]] -> Util.walkParsed[MT](Set(scopedParent), initialScope, soql)
+      Option.empty[types.ScopedResourceName[MT]] -> Util.walkParsed[MT](Set(scopedParent).filterNot(Util.isSpecialTableName), initialScope, soql)
     )
-    new QueryGraph[MT](None, buildGraph(Some(scopedParent), buildGraph(None, seed)))
+    new QueryGraph[MT](None, buildGraph(None, buildGraph(None, seed)))
```

Comment:
This line was just wrong; the root of a parentedGraph based on anonymous soql _is_ the anonymous soql (== None)